privaterelay/settings.py

```diff
@@ -105,10 +105,11 @@
     "https://location.services.mozilla.com",
     BASKET_ORIGIN,
 )
-CSP_DEFAULT_SRC = ("'self'",)
+CSP_DEFAULT_SRC = ("'self'", "https://js.stripe.com/")
```

Comment:
I don't think `CSP_DEFAULT_SRC` needs to be changed, we have more specific directives that will override it.